apps/web/src/lang/fr.json

```diff
@@ -6646,4 +6646,4 @@
     "defaultMessage": "Mes candidatures",
     "description": "Title for the list of a users applications"
   }
-}
\ No newline at end of file
+}
```

Comment:
I've noticed this a few times. Is there a reason `.editconfig` is being ignored or is it because the file is being generated without a new line via the cli?

https://github.com/GCTC-NTGC/gc-digital-talent/blob/149f115ad0fbcbb43caf756528c831edcc9cc221/.editorconfig#L8

https://github.com/GCTC-NTGC/gc-digital-talent/blob/149f115ad0fbcbb43caf756528c831edcc9cc221/packages/i18n/src/cli.ts#L103-L106

Reply:
Pretty sure that was what was happening. [replace os.EOL with \n to avoid cross platform issues](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5707/commits/c7ff1ad26463f63006d5d62c3c3de80791eca087) should fix it.